homeassistant/components/home_connect/entity.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, device: HomeConnectDevice, desc: str) -> None:
         self.device = device
         self.desc = desc
         self._name = f"{self.device.appliance.name} {desc}"
+        self.device.entities.append(self)
```

Comment:
We should not store the entity in a common container on the device. The device should not know about any entities.